openapi/giphy/types.bal

```diff
@@ -0,0 +1,135 @@
+public type Response record {
```

Comment:
Missing license header

Reply:
Fixed in https://github.com/ballerina-platform/ballerinax-openapi-connectors/commit/fcd3580d6892ae0cfbfa1e9449cc814999a11f4b